build/win32.js

```diff
@@ -1,30 +1,32 @@
-var childProcess, tableParser, _;
+var childProcess, path, _;
 
 childProcess = require('child_process');
 
 _ = require('lodash');
 
-tableParser = require('table-parser');
+path = require('path');
 
 exports.list = function(callback) {
-  return childProcess.exec('wmic diskdrive get DeviceID, Caption, Size', {}, function(error, stdout, stderr) {
-    var result;
+  var script;
+  script = path.join(__dirname, '..', '/scripts/win_drives.vbs');
```

Comment:
Can you split `/scripts/win_drives.vbs` into separate arguments to `path.join`? The purpose is to not hardcode any file path delimiter, and let `path.join` use the appropriate one.